content/using-sagemaker-managed-spot-training/notebooks/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Running Notebooks"
+chapter: false
+weight: 20
+---
+
+The GitHub Repository cloned to the Amazon SageMaker Notebook Instance contains sample notebooks that demonstrate leveraging Managed Spot Training with various frameworks. You can view these notebooks directly through the following link: [EC2 Spot Labs](https://github.com/awslabs/ec2-spot-labs/tree/master/ec2-spot-sagemaker-managed-spot-training). 
+
+If you're familiar with SageMaker Notebook Instances and Jupyter Lab then you can use the following steps to skip ahead and work through any example notebooks you're interested in. However, the following sections will provide more details on these notebooks, and how to configure Amazon SageMaker to leverage Spot Instances when executing training jobs.
+
+1. Logon to the AWS Console.
+
+2. Browse to the Amazon SageMaker Notebook Instances Console.
+
+3. Open JupyterLab.
+
+4. Open the Notebook you'd like to run.
+
+5. Execute each step in the notebook in order.
```

Comment:
I still need to go through Sagemaker Studio and Debugger, but perhaps on a second iteration it will be good to add this case as a new item within the menu just right after the "Running Notebooks" section.

https://github.com/awslabs/amazon-sagemaker-examples/tree/master/sagemaker-debugger/mxnet_spot_training

The cool stuff for Spot, is that sometimes during debugging you may end up trying different things and end up costing a bit, by using spot you might still keep the sessions down to a min cost... nothing that you don't know here :)

Personally I would not hold the release of this workshop, and instead add the debugger module on a second iteration.

Reply:
Thinking a bit more about it, for a second iteration, perhaps we can add a description of why Spot is great on AI workloads and the areas where it adds loads of value and can make the process of finding the right model + parameters very expensive.  The other use case that I reckon may worth adding and highlighting as a module: Hyper-parameter tuning . https://docs.aws.amazon.com/sagemaker/latest/dg/API_CreateHyperParameterTuningJob.html . Perhaps given this training has done something similar adapting the following example : https://github.com/awslabs/amazon-sagemaker-examples/blob/master/hyperparameter_tuning/image_classification_early_stopping/hpo_image_classification_early_stopping.ipynb to include spot in the hyper-parameter tunning section

Reply:
Good call, can add this to the next iteration. It'll be great to run this a few times and see what customers think and ask about as well.